.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+Please include a summary of the change and which issue is fixed or which feature is added.
+
+- [ ] Issue 1 fixed
+- [ ] Issue 2 fixed
+- [ ] Feature 1 added
+- [ ] Feature 2 added
+
+Fixes # (issue)
+
+# How to test this?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce if there is no integration test added with this PR. Please also list any relevant details for your test configuration
+
+"""
+cmake ..
+make -j8
+nrnivmodl mod
+./bin/nrnivmodl-core mod
+./x86_64/special script.py
+./x86_64/special-core --tstop=10 --datpath=coredat
+"""
+
+**Test Configuration**:
+* Compiler:
+* Backend:
+* OS:
+* Options:
+
+# Use certain branches for the SimulationStack CI
+
+CI_BRANCHES:NEURON_BRANCH=master,SPACK_BRANCH=master,PYNEURODAMUS_BRANCH=master,REPORTINGLIB_BRANCH=master,PYNEURODAMUS_BRANCH=master
```

Comment:
For CI not to fail I thought we should have by default: 
```
#default setting
CI_BRANCHES:NEURON_BRANCH=master
```

Also, the other ones are more for internal purposes? Regular users may/should not be aware.

Reply:
You're right that only `NEURON_BRANCH=master` is needed by default. I just added all the possible options in case someone wants to use them in the future. I will remove them though since most of the time they are unnecessary.